Add footer, code of conduct, and privacy policy

Pete-Fowler opened the pull request.

Pete-Fowler committed:
Add HTML to code, add content to privacy policy

Pete-Fowler committed:
Add HTML to privacy policy

Pete-Fowler committed:
Add footer, position RD logo

Pete-Fowler committed:
Add and position footer skyline

Pete-Fowler committed:
Add and position links. Need routes

Pete-Fowler committed:
Add proper link / routing

Need style for code of conduct and privacy policy

Pete-Fowler committed:
Add formatting to conduct code and privacy policy

Pete-Fowler committed:
Add link home on RD footer logo

Pete-Fowler commented:
FYI I did some very slight rewording of Tito's privacy policy, to just make it work without creating the table they had. Nothing of substance was changed.

JeffBaumgardt reviewed (approved):
Looking good thus far.

I'm not sure the import of styles in your footer are going to work. Remix doesn't resolve css imports like webpack does with Create React App. In Remix you import the styles but have to add them to a special named export of `export function links()` 

We're still nailing down exactly what we are going to do with styles in general. 

But for additional information about how Remix does styles a bit different check out - https://remix.run/docs/en/v1/guides/styling

Pete-Fowler commented:
I see. Do you think adding the following - the same as in the header - will make it work properly once deployed and not just on my computer? I added this and updated this remote branch.
```
export function links() {
  return [
    {
      rel: "stylesheet",
      href: styles,
    },
  ];
}
```

Pete-Fowler committed:
Add export function links() to footer

Pete-Fowler committed:
Fix erroneous underscore in css rule
